src/NumericalAlgorithms/LinearOperators/PowerMonitors.cpp

```diff
@@ -17,6 +17,39 @@
 
 namespace PowerMonitors {
 
+double detail::relative_truncation_error_impl(
+    const DataVector& input_power_monitors, const size_t upperBound) {
```

Comment:
how about `upperBound` -> `num_modes`

Reply:
I wanted to use this function in the tests. Not sure how to call it if its not in the hpp file

Reply:
You can also make it a proper public function if you prefer: no detail namespace or impl suffix, just a regular overload of `relative_truncation_error` that works in one dimension.